elements/lisk-tree/test/merkle_tree/utils.spec.ts

```diff
@@ -54,4 +54,25 @@ describe('utils', () => {
 			}
 		});
 	});
+
+	describe('calculateMerkleRootWithLeaves', () => {
+		describe('should generate correct merkle root info', () => {
+			for (const test of fixture.testCases) {
+				describe(test.description, () => {
+					const transactionIds = test.input.transactionIds.map(hexString =>
+						Buffer.from(hexString, 'hex'),
+					);
+					const tree = new MerkleTree();
```

Comment:
Instead of creating merkle tree, fixture has `transactionMerkleRoot` to compare with